.github/CODEOWNERS

```diff
@@ -2,6 +2,12 @@
 # For more information on CODEOWNERS file see : https://help.github.com/en/articles/about-code-owners
 
 # For the entire repository 
-* @dotnet/mlnet-core
- 
+*	@dotnet/mlnet-core
 
+# Reviewers for files owned by AutoML team
+src\Microsoft.ML.AutoML	@dotnet/mlnet-automl
+src\mlnet	@dotnet/mlnet-automl
```

Comment:
This folder doesn't exist.

Same for `test\mlnet.Tests`

Reply:
@harshithapv can you please remove these folders? @Dmitry-A can you please confirm if there any other folders you would want to own?

Reply:
mlnet stuff is currently in a feature branch but we will be merging it sometime soon, so I thought to add them here ahead of time.  We can either do that or add them later, either way works for me.

Reply:
Lets add it after we merge. It does not make sense to add path to a folder that does not exist.